docs/api/fhir-api-specification.md

```diff
@@ -3011,3 +3208,421 @@ GET `[base]/Patient?_count=2`
   </div>
 </details>
 
+## Transactions
+The API supports performing several actions as a single atomic "transaction" for which all of the individual changes succeed or fail together. 
+
+<a name="transaction-post"/>
+
+### POST `[base]`
+To perform a transaction, POST a FHIR [Bundle](https://www.hl7.org/fhir/bundle.html) resource to `[base]`. The Bundle must have `Bundle.type` set to `transaction`. Currently a transaction may only be used to create monitorees and lab results via the FHIR Patient and Observations resources, respectively. Additionally, if a transaction is used to create an Observation, that Observation must reference a Patient being created by the same transaction. Each entry in the `Bundle.entry` array should contain the following fields:
```

Comment:
Yeah I was just a bit wary of advertising it explicitly, since no one has explicitly been asking for it, but you're right, since we are allowing it, and since I think it's usefulness is pretty self explanatory, I will advertise it more explicitly. I might be being overly cautious, but how do you feel about 50 as a default max batch size? And then if we hit a situation where people are telling us they need larger batches, it feels easier to raise the maximum than to lower it if we find 100 to be problematic in any way.

Definitely a good call to advertise this though, at the very least it usually takes ~150 ms to create a single patient, but to bulk create 50 patients only took 4000 ms, so it's almost half as long per patient.

Reply:
I'm good with making it 50. That's still large enough that it's a lot more useful than individual create calls.

Reply:
Done in https://github.com/SaraAlert/SaraAlert/pull/870/commits/dd09efe93d1e63df21acf5daf7509a0a1c1b6ec6.